packages/components/docs/fileAttachment.md

```diff
@@ -0,0 +1,64 @@
+# FileAttachmentForm
+
+Rendering FileAttachmentForm component allows users to select files to be uploaded to the server
+via file selection or drag & drop.
+Users can then utilize our WebDAV API to upload or retrieve existing uploads -
+example [here](./webdav.md).
+
+## [FileAttachmentForm](../src/public/files/FileAttachmentForm.tsx#L71)
+```tsx
+
+//imports
+import React, { FC, memo, useCallback } from 'react';
+import { Panel } from 'react-bootstrap';
```

Comment:
can this import be removed? looks like the usage was removed per Nick's suggestion

Reply:
done